From 93f8c315d2f5b3f1e611fe67dec7d1a41ba615d7 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Thu, 20 May 2021 05:16:39 +0300
Subject: [PATCH] [SW-0] greco: security: add unused attribute

Add unused attribute in preboot to use for base
api preparation patch.

Change-Id: I6e5ab7fe6158e02c3004bb71a4ee1a0d9b19a367
Signed-off-by: Karthikeyan K <karthikk@habana.ai>
---

diff --git a/include/preboot.h b/include/preboot.h
index 00bdd84..bdd3810 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -52,6 +52,9 @@
 #ifndef __aligned
 #define __aligned(x)	__attribute__((aligned(x)))
 #endif
+#ifndef __unused
+#define __unused	__attribute__((__unused__))
+#endif
 
 enum hl_rst_src {
 	HL_COLD_RST,
